### ISSUE TITLE
[FEAT-REQ] Ability to use multiple trees simultaneously

### ISSUE THREAD

**Issue by StasSharp** [JairajJangle/react-native-tree-multi-select#78]
**Is your feature request related to a problem? Please describe.**
First of all, thanks for this truly amazing library!

My problem is that I need to use two trees in one view with different data sets, and since the library is using Zustand with a global store, both trees were showing the same set of values. I was able to overcome it with a dirty hack, but it would be great if there was the ability to use multiple trees in one view.

**Describe the solution you'd like**
The best solution would be to be able to have any amount of TreeViewes hosted on the same page with no data duplication. Something like:
```typescript
<TreeView data={data1} />  
<TreeView data={data2} />
```
showing different data.

**Describe alternatives you've considered**
I ended up duplicating the store and duplicating the TreeView component (to have "TreeView2") for now, but it's a very dirty solution, it would be great if the library supported it.

**Screenshots or screen records**
NA

**Additional context**
NA

**Would you like to work on this feature?**
Unfortunately, I'm not good enough with React to be able to work on it

**Comment by JarvisMarkDown** [comment]
Why hasn't this problem been solved?

**Comment by JairajJangle** [comment]
@JarvisMarkDown Hi there! 👋

Thanks for pointing this out. Just to clarify, this library is open-source and heavily relies on contributions from the developer community. If you'd like to jump in and help, that would be amazing! I'm always happy to collaborate and guide anyone interested. 😊

That said, the fix is already on the way, and I appreciate your patience as it's being worked on. Let me know if you have any other thoughts or suggestions!

**Comment by JairajJangle** [comment]
Hey @StasSharp! 👋

Thank you for your kind words and for sharing your detailed use case! 😊  

I'm excited to let you know that the feature you requested—support for multiple `TreeView` components with separate data—is now available in [v1.6.0-beta.1](https://github.com/JairajJangle/react-native-tree-multi-select/releases/tag/v1.6.0-beta.1) and also on npm: [v1.6.0-beta.1](https://www.npmjs.com/package/react-native-tree-multi-select/v/1.6.0-beta.1). 🎉  

With this release, you can now use multiple `TreeView` components on the same page without data duplication, just as you described:  

```tsx
<TreeView data={data1} />  
<TreeView data={data2} />
```  

Feel free to give it a try and let me know if you encounter any issues or have additional feedback. I’ll be closing this issue for now, but don’t hesitate to reach out if anything comes up. 🙏  

Thanks again for your feature request and valuable feedback! Also, just a heads-up that I’m planning to promote the beta to the main release in about a week or so.
